pom.xml

```diff
@@ -39,7 +39,8 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <logback.classic.version>1.2.3</logback.classic.version>
-
+        <maven.enforcer.java>1.8</maven.enforcer.java>
+        <maven.enforcer.mvn>3.5.1</maven.enforcer.mvn>
```

Comment:
Are these the only versions supported by Corfu?

Reply:
I think other versions might break with the way we process annotations.